onnx/backend/test/cpp/driver_test.cc

```diff
@@ -237,25 +248,24 @@ class ONNXCppDriverTest
           result.buffer = (onnxPointer)raw_data.data();
           result_descriptor.emplace_back(std::move(result));
         }
-        ASSERT_EQ(
+        ASSERT_MEMORYSAFE_SUCCESS(
             lib.onnxSetGraphIO(
                 graph,
                 input_descriptor.size(),
                 input_descriptor.data(),
                 result_descriptor.size(),
                 result_descriptor.data()),
-            ONNXIFI_STATUS_SUCCESS);
+            true);
 
         onnxMemoryFenceV1 inputFence, outputFence;
         inputFence.tag = ONNXIFI_TAG_MEMORY_FENCE_V1;
         outputFence.tag = ONNXIFI_TAG_MEMORY_FENCE_V1;
         inputFence.type = ONNXIFI_SYNCHRONIZATION_EVENT;
         outputFence.type = ONNXIFI_SYNCHRONIZATION_EVENT;
 
-        ASSERT_EQ(
-            lib.onnxRunGraph(graph, &inputFence, &outputFence),
-            ONNXIFI_STATUS_SUCCESS);
-        ASSERT_EQ(lib.onnxWaitEvent(outputFence.event), ONNXIFI_STATUS_SUCCESS);
+        ASSERT_MEMORYSAFE_SUCCESS(
+            lib.onnxRunGraph(graph, &inputFence, &outputFence), true);
```

Comment:
I might be missing something, but how does the whole workflow work without actually signaling on inputFence?

Reply:
I think you are right about this! I wonder why some of the results are correct...
It might because I didnot initialie the event at all, and therefore both output.event and input.event are set to signaled by default.
Let me fix this, thanks!